shared/common-adapters/text.meta.desktop.js

```diff
@@ -238,6 +238,19 @@ const metaData: {[key: TextType]: MetaType} = {
     fontSize: 11,
     styleOverride: globalStyles.fontRegular,
   },
+  // Body tiny
+  BodyTiny: {
```

Comment:
?

Reply:
We needed a smaller text style to match the design. Is this not the way to do it?

Reply:
In those cases I usually use the closest `TextType` and apply the necessary styles adhoc. I think these are meant to correspond with what's in the font styleguide